### PR TITLE
fix(insight-tooltip): Add commas to insight tooltip count

### PR DESCRIPTION
**Pull request by liyiy**
## Changes

before:
Screen Shot 2022-05-10 at 3 01 04 PM

after:
Screen Shot 2022-05-10 at 2 57 50 PM

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

**Comment by liyiy** [comment]
quick fix? 😄

**Review by clarkus — COMMENTED** [review]
Looks like a good change to me. Can we change the precision to 2 decimal places instead of 3? I just don't know that 3 decimal precision is going to be valuable in this scenario. I'd rather we have that extra character of space for the label. 

@marcushyett-ph any thoughts on precision with numeric formatting?

**Review by clarkus — APPROVED** [review]

**Comment by liyiy** [comment]
Screen Shot 2022-05-10 at 3 10 56 PM

File: frontend/src/lib/utils.tsx [hunk 1]
```diff
@@ -1104,7 +1104,7 @@ export function maybeAddCommasToInteger(value: any): any {
     if (!isNumber) {
         return value
     }
-    const internationalNumberFormat = new Intl.NumberFormat('en-US')
+    const internationalNumberFormat = new Intl.NumberFormat('en-US', { maximumFractionDigits: 2 })
```
Comment (Twixes) [inline frontend/src/lib/utils.tsx:1107]: Oh, didn't know we had `maybeAddCommasToInteger()`. I've been using `humanFriendlyNumber()` recently. Perhaps we can unify those.

**Pull request merged**